src/components/ui-geo-map/ui-geo-map.tsx

```diff
@@ -13,22 +13,35 @@ export class UiGeoMap {
   @Element() private el!: HTMLElement;
 
   private map?: google.maps.Map;
+  private marker?: google.maps.Marker;
 
   public componentDidLoad() {
+    if (!Build.isBrowser) {
+      return;
+    }
     const m = this.el.shadowRoot!.querySelector("#map");
     this.map = new google.maps.Map(m as HTMLElement, {
       center: { lat: 38.8899, lng: -77.0091 },
-      zoom: 12,
+      zoom: 16,
       fullscreenControl: false,
       mapTypeControl: false,
       streetViewControl: false,
+      scrollwheel: false,
+    });
+
+    this.marker = new google.maps.Marker({
+      icon: "/images/pin-pizza@1x.png",
```

Comment:
let's make this not transparent 

![image](https://user-images.githubusercontent.com/700444/97742543-a8dc5180-1aa1-11eb-9f80-f170e50f98a4.png)

Reply:
Should be able to add a white background behind the transparency -- Sarah should be able to export a new asset.